core/src/main/java/org/elasticsearch/action/update/UpdateResponse.java

```diff
@@ -65,22 +76,17 @@ public GetResult getGetResult() {
      * Returns true if document was created due to an UPSERT operation
      */
     public boolean isCreated() {
```

Comment:
I'm tempted to remove `isCreated` and just use `getOperation` everywhere. Might make more sense to do in a followup PR because it is a lot of small mechanical changes.